xla/stream_executor/blas.h

```diff
@@ -569,7 +592,8 @@ class BlasSupport {
       uint64_t m, uint64 n, uint64 k, blas::DataType dtype, const void *alpha, \
       const DeviceMemoryBase &a, int lda, const DeviceMemoryBase &b, int ldb,  \
       const void *beta, DeviceMemoryBase *c, int ldc,                          \
-      const NumericOptions &numeric_options) override;                         \
+      const NumericOptions &numeric_options,                                   \
+      blas::CallContext context = blas::CallContext::kNone) override;                                     \
```

Comment:
nit: the formatting is off here and below.

Reply:
Thank you for fixing this. I think you currently cannot see this on PRs and I have provided this feedback internally. We should be able to get this functionality back.

Reply:
@sherhut  Yes, it would be nice to see formatting errors in the PR/CI itself.

For now, how can I see where the formatting is an issue?

Thanks!

Reply:
Note for the future - Copybara will format the code automatically when we import, so I think small formatting errors like this are fine. I've seen a few edge cases on PRs (e.g. where it's impossible to automatically format the line to be less than 80 characters), but in general formatting shouldn't be an issue

Reply:
If the only issue is the formatting problem that Stephan is talking about then I think it should be ok - I'll wait for him to take another look to be sure as I don't have the full context here. I realize I also didn't answer your original question - using `clang-format -style=google -dry-run` on files you edit should be very close/identical to what our internal formatter will want, although I expect it'll be rare that it will catch issues that couldn't be automatically resolved on import

Reply:
The following test is broken for this PR in the internal CI:
https://github.com/openxla/xla/blob/main/xla/service/xla_aot_compile_gpu_test.cc#L130

Can you please double-check. Thank you!